smt/surrogate_models/krg_based.py

```diff
@@ -78,6 +81,12 @@ def _initialize(self):
             ),
             desc="Correlation function type",
         )
+        declare(
+            "pow_exp_power",
+            3.0,
```

Comment:
Can you initialize to a value other than 3 ? Let's say 1.5 ?

Reply:
Yes! sorry I tested different default values locally but forgot to change it back.